src/governance.rst

```diff
@@ -0,0 +1,173 @@
+
+Voting items are labeled as either **standard** or **sensitive**.
+Standard items are ones where public record and discourse and record is
+preferable. Sensitive voting items are ones where the results of the
+vote should remain private to the voters after the vote has occurred.
+Sensitive votes should take place on
+`the Helios voting system <https://vote.heliosvoting.org/>`_ in order
+retain anonymity.
+
+The default voting period is 1 week (7 days). This may be modified at
+the time when a vote is called, but may never be less than 24 hrs.
+
+:Nominate new member of staged-recipes: The proposer must provide
+    a brief justification as to why the new member is desirable or needed.
+
+    * Sensitive
+    * 50% Majority to pass
+
+:Nominate new member of core: The proposer must provide
+    a sufficient justification as to why the nominee should be welcomed
+    into core. It is necessary that all nominees are current and active
+    members of staged-recipes. Nominees to core must have been members
+    of staged-recipes for at least 1 month (4 weeks) prior to this vote.
+
+    * Sensitive
+    * 66.7% Majority to pass
+
+:Sub-team Formation: Proposers must specify the name, role & responsibility,
+    members, and charter (dynamic or static) of any new sub-teams.
+
+    * Standard
+    * 50% Majority to pass
+
+:Sub-team Dissolution: Proposers must specify the name and justification
+    for why a sub-team should be dissolved.
+
+    * Standard
+    * 50% Majority to pass
+
+:Lock an Issue, Pull Request, Thread: Occasionally, discussions become
+    toxic and antithetical to the goal of fostering the conda-forge
+    community. Members of core have the right to lock the thread in an
+    "ask for forgiveness and not for permission" way so bad situations
+    are handled quickly. The lock must be justified in the thread itself
+    with a text explaining the reasons for locking and how the participants
+    may can contest it.
+
+    * Standard
+    * No need for voting to lock a thread
+
+:Block a Contributor: In extreme cases, such as repeated harassment,
+    it may become necessary to block a user completely from participating
+    in all conda-forge activities. This should not be done lightly,
+    but it may be necessary to do so expediently. Shorter voting periods
+    (such as 24 hrs) are to be expected. The proposer of the block
+    must provide ample justification as to why this is needed.
+
+    * Sensitive
+    * 60% Majority to pass
+
+
+:Remove member of staged-recipes: The proposer must provide
+    a justification as to why the member of staged recipes
+    should be removed.
+
+    * Sensitive
+    * 66.7% Majority to pass
+
+:Remove member of core: The proposer must provide
+    an overwhelming justification as to why the member core
+    should be removed.
+
+    * Sensitive
+    * 75% Majority to pass
+
+All other voting items are considered to be standard and require a 50%
+majority to pass.
```

Comment:
As discussed in our meeting earlier, we may want a provision for amending this document (as it is a living document). It sounds like the text here mostly matches what we want. We could add a note about amending the document here or perhaps in a nearby section.